shared/src/main/kotlin/com/egm/stellio/shared/util/NgsiLdParsingUtils.kt

```diff
@@ -220,7 +232,7 @@ object NgsiLdParsingUtils {
     }
 
     fun extractShortTypeFromPayload(payload: Map<String, Any>): String =
-        // TODO is it always after a '/' ? can't it be after a '#' ? (https://redmine.eglobalmark.com/issues/852)
+    // TODO is it always after a '/' ? can't it be after a '#' ? (https://redmine.eglobalmark.com/issues/852)
```

Comment:
Hmm, why isn't it aligned with the comment below (and the line after)?

Reply:
In my opinion this is caused by the function without braces, with braces it is correctly formatted.
Why is this happening ? I don't know. I couldn't find a way to arrange that.
Without the comments, it formats like that : 
```kotlin
fun extractShortTypeFromPayload(payload: Map<String, Any>): String =
        (payload["@type"] as List<String>)[0].substringAfterLast("/").substringAfterLast("#")
```
and with a multi line comment like that : 
```kotlin
fun extractShortTypeFromPayload(payload: Map<String, Any>): String =
    /* 
     * TODO do a clean implementation using info from @context
     * TODO is it always after a '/' ? can't it be after a '#' ? (https://redmine.eglobalmark.com/issues/852)
     */
    (payload["@type"] as List<String>)[0].substringAfterLast("/").substringAfterLast("#")
```
I think a common rule is to use multi line comment instead of having 2 separate comments. Should I replace it by a multi-line comment ?

Reply:
OK